NAMESPACE

```diff
@@ -1,50 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
-export(check_dhis2_attributes)
-export(connect_to_server)
-export(detect_separator)
-export(dhis2_subset_fields)
-export(dhis2_subset_records)
-export(fetch_data_from_query)
-export(fingertips_subset_columns)
-export(fingertips_subset_rows)
-export(get_base_name)
-export(get_data_elements)
-export(get_data_sets)
-export(get_extension)
-export(get_fingertips_metadata)
-export(get_ind_id_from_domain_id)
-export(get_ind_id_from_domain_name)
-export(get_ind_id_from_ind_name)
-export(get_ind_id_from_profile)
-export(get_organisation_units)
-export(get_profile_name)
-export(get_read_file_args)
-export(get_read_fingertips_args)
-export(get_relevant_data_elt_group)
-export(get_relevant_dataset)
-export(get_relevant_organisation_unit)
-export(identify_table_name)
-export(import_redcap_data)
-export(read_credentials)
-export(read_from_dhis2)
-export(read_from_file)
-export(read_from_fingertips)
-export(read_from_redcap)
-export(read_multiple_files)
+export(get_dhis2_attributes)
```

Comment:
When fetching data from DHIS2, users can either provide an id for a dataElement, dataSet, dataElementGroup, organisationUnits or a combination of these. 
To list all dataElements,  or dataSet, or dataElementGroup, or organisationUnits, users can use the  `get_dhis2_attributes()` function. This will help in confirming whether the chosen id is part of the system and also to have an overview of all attributes IDs in the system.